providers/bnxt_re/verbs.c

```diff
@@ -222,6 +225,103 @@ struct ibv_cq *bnxt_re_create_cq(struct ibv_context *ibvctx, int ncqe,
 	return NULL;
 }
 
+static int bnxt_re_poll_kernel_cq(struct bnxt_re_cq *cq)
+{
+	struct ibv_wc tmp_wc;
+	int rc;
+
+	rc = ibv_cmd_poll_cq(&cq->ibvcq, 1, &tmp_wc);
+	if (rc)
+		fprintf(stderr, "ibv_cmd_poll_cq failed: %d\n", rc);
```

Comment:
No prints in libraries.

I fixed it locally and removed whole bnxt_re_poll_kernel_cq function as it has only one line ibv_cmd_poll_cq(...)
with return value which no one is interested.